src/job/mod.rs

```diff
@@ -95,6 +95,8 @@ pub async fn init() -> Result<(), anyhow::Error> {
         tokio::task::spawn(async move { telemetry::run().await });
     }
 
+    tokio::task::spawn(async move { usage::run().await });
```

Comment:
@hengfeiyang should it run on all nodes? or I should leave compactor roles?

Reply:
What is this job? accept usage event and send to ingester? Which node has the usage event? all the nodes or only some nodes?

Reply:
i am thinking we should run it on every node.